README.md

```diff
@@ -3,7 +3,17 @@
 Basic statistics and data analysis with Java
 
 ## Dependencies
-* Jackson Databind <https://github.com/FasterXML/jackson>
-* Apache Commons CSV <https://commons.apache.org/proper/commons-csv/>
-* Tablesaw Java Dataframe and Visualization Library <https://jtablesaw.github.io/tablesaw/>
+
```

Comment:
I like what you did with the readme. let's keep this version.

Reply:
ok cool